docs/guide.md

```diff
@@ -0,0 +1,104 @@
+# Install openebs-monitoring Chart in already installed prometheus-stack
+
+### Prerequisite
+
+- Already installed prometheus-operator stack
+- OpenEBS installed
+
+---
+**NOTE**
+
+To allow Prometheus to discover all `PodMonitors/ServiceMonitors`, without applying label filtering, you have to set `prometheus.prometheusSpec.podMonitorSelectorNilUsesHelmValues` and `prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues` to `false`
+
+---
+
+## Step by Step Process
+
+1. **Update values.yaml**
+   
+   -  Disable dependent chart(kube-prometheus-stack) by updating `kube-prometheus-stack.enabled` to `false`
+   -  Set `customPrometheusStack.enabled` to `true` 
```

Comment:
If incorporating above name change then make a change in all the corresponding places.